Runtime/Scripts/RGBotServerListener.cs

```diff
@@ -498,18 +498,25 @@ private void FixedUpdate()
             }
         }
 
-        
-        
         /**
          * Gets the entire game state by searching for all RGState game objects and gather their
          * states.
          */
         private Dictionary<string, RGStateEntity> GetGameState()
         {
             var overlayAgent = this.gameObject.GetComponent<RGEntity>();
-
             var statefulObjects = FindObjectsOfType<MonoBehaviour>(true).OfType<IRGState>();
             var fullGameState = new Dictionary<string, RGStateEntity>();
+            var duplicatedFields = new List<string>()
```

Comment:
Let's define this outside the method so it doesn't get re-allocated on every call

Reply:
Done 👍